en/components/general/update-guide.md

```diff
@@ -51,6 +51,35 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 14.1.x to 14.2.x
+### General
+- **Breaking Change** - All **igx** prefixes are replaced with **ig** prefixes. For example, **igx-**typography is now **ig-**typography.
+
+- Add this line in your `angular.json` file:
+```
+"stylePreprocessorOptions": {
+    "includePaths": ["node_modules"]
+}
+```
```

Comment:
Where exactly in my angular.json file should I add this? Further clarification is needed.